openreview/conference/templates/areachairWebfield.js

```diff
@@ -612,6 +635,62 @@ var renderTableRows = function(rows, container) {
   $(container).append(tableHtml);
 }
 
+var postRenderTable = function(rows) {
+
+  currentRankings = [];
+  rows.forEach(function(row) {
+    if(row[4].ranking && row[4].ranking.tag !== 'No Ranking') {
```

Comment:
I've decided to leave 'No Ranking' always visible for the user to unselect a number.